### PR TITLE
3p-lib updates

### PR DESCRIPTION
**Pull request by marchcat**
- tinyexr to v1.0.9 - secondlife/3p-tinyexr#5;
- freetype to v2.13.3 - secondlife/3p-freetype#14;
- boost to v1.86.0 - secondlife/3p-boost#13;
- (dependency) secondlife/3p-colladadom#11.

### CODE REVIEW

File: autobuild.xml [hunk 1]
```diff
@@ -99,18 +99,28 @@
       </map>
       <key>boost</key>
       <map>
+        <key>copyright</key>
```
Comment (Ansariel) [inline autobuild.xml:102]: Pretty sure the next one doing library updates is going to shuffle all these keys to the end again... :joy:

Comment (marchcat) [reply autobuild.xml:102]: It depends on the autobuild version, but yes, it's a known problem.

**Comment by RyeMutt** [comment]
You'll also need to update boost in https://github.com/secondlife/3p-colladadom and rebuild that

**Comment by marchcat** [comment]
> You'll also need to update boost in https://github.com/secondlife/3p-colladadom and rebuild that

Right, working on that now.

**Review by akleshchev — APPROVED** [review]

**Pull request merged**